### PR TITLE
Secure password retrieval via environment variable

### PR DESCRIPTION
**Pull request by avilagaston9**
This PR modifies the multi_node_setup.sh script to retrieve the validator password from the environment variable PASSWORD. 'password' is set as the default.

### CODE REVIEW

File: multi_node_setup.sh
Comment (JulianGCalderon) [inline multi_node_setup.sh:1]: The file has no execution permissions. Run:
```
chmod +x multi_node_setup.sh
```

**Review by JulianGCalderon — CHANGES_REQUESTED** [review]
Seems redundant to have a variable with the same value as an ENV var. I'd change the script to use only the ENV var

**Review by JulianGCalderon — DISMISSED** [review]
LGTM

**Review by tomyrd — CHANGES_REQUESTED** [review]
We should mention the env var in the README multi node tutorial

**Review by JulianGCalderon — APPROVED** [review]
LGTM

**Review by tomyrd — APPROVED** [review]
LGTM

**Pull request merged**